### PR TITLE
Add Maya swatch classification to appleseed texture shaders.

### PR DESCRIPTION
**Pull request by est77**
- Added Maya's swatch classification for textures.
- Added textureToSuface shader for Maya texture swatches.

### CODE REVIEW

File: src/appleseed.shaders/src/maya/as_maya_texture2Surface.osl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// THE SOFTWARE.
+//
+
+surface as_maya_texture2Surface
+(
+    color in_color = 1,
+    float in_scalar = 1
+)
+{
+    Ci = in_color * in_scalar * emission();
```
Comment (luisbarrancos) [inline src/appleseed.shaders/src/maya/as_maya_texture2Surface.osl]: @est77 wouldn't it make more sense to run this only for *light* rays?

Comment (est77) [inline src/appleseed.shaders/src/maya/as_maya_texture2Surface.osl]: for this specific case, I think it's not very important. 
We can merge it now and improve it later if it really makes a difference.

Comment (luisbarrancos) [inline src/appleseed.shaders/src/maya/as_maya_texture2Surface.osl]: @est77 can i merge?

Comment (est77) [inline src/appleseed.shaders/src/maya/as_maya_texture2Surface.osl]: Yes. I thought you were going to merge it. 
No need to ask really. If you think a PR is ok you can merge it directly.

**Pull request merged**